recipe/meta.yaml

```diff
@@ -1,52 +1,48 @@
 {% set name = "assimp" %}
-{% set version = "5.0.1" %}
+{% set version = "5.4.3" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
-  url: https://github.com/assimp/assimp/archive/v{{ version }}.tar.gz
-  sha256: 11310ec1f2ad2cd46b95ba88faca8f7aaa1efe9aa12605c55e3de2b977b3dbfc
-  patches:
-    - patch.diff  # [linux]
-    - fix_cmake_config.diff
-    - do_not_export_debug_symbols_gcc.patch
+  url: https://github.com/{{ name }}/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 66dfbaee288f2bc43172440a55d0235dfc7bf885dda6435c038e8000e79582cb
 
 build:
-  number: 7
+  number: 0
   skip: True  # [s390x]
   run_exports:
-    - {{ pin_subpackage(name, max_pin='x.x') }}
+    - {{ pin_subpackage(name, max_pin='x.x.x') }}
 
 requirements:
   build:
     - {{ compiler('cxx') }}
     - {{ compiler('c') }}
     - cmake
     - make  # [unix]
-    - patch  # [unix]
-    - m2-patch  # [win]
   host:
-    - boost-cpp 1.82
-    - zlib
+    - boost-cpp {{ boost_cpp }}
```

Comment:
I don't see boost referenced, can you try removing it?

Reply:
Thanks for catching it. I removed `boost-cpp`. It works fine.